### ISSUE TITLE
Missing node labels in SVG export of graph

### ISSUE THREAD

**Issue by dhimmel** [greenelab/connectivity-search-frontend#155]
For [this query](https://het.io/search/?source=2170&target=13008&metapaths=DaG%3CrGpBP%2CDaGcGpBP%2CDaGiGpBP%2CDaGpBP%2CDaGr%3EGpBP%2CDdG%3CrGpBP%2CDdGcGpBP%2CDdGiGpBP%2CDdGpBP%2CDdGr%3EGpBP%2CDlAdGpBP%2CDlAeGpBP%2CDlAuGpBP%2CDpCbGpBP%2CDpCdGpBP%2CDpCuGpBP%2CDrDaGpBP%2CDrDdGpBP%2CDrDuGpBP%2CDtCbGpBP%2CDtCdGpBP%2CDtCuGpBP%2CDuG%3CrGpBP%2CDuGcGpBP%2CDuGiGpBP%2CDuGpBP%2CDuGr%3EGpBP&complete=) I created a graph with the top 4 paths:

> ![image](https://user-images.githubusercontent.com/1117703/68799490-425aeb00-0626-11ea-8c50-7580f3b50689.png)

When viewing the downloaded SVG in Chrome, there are no node labels:

> ![image](https://user-images.githubusercontent.com/1117703/68799571-64546d80-0626-11ea-8430-1bd15a42403f.png)

Here's the source of `graph_multiple-sclero_retina-layer-fo.svg`:

<details>

```svg
<svg xmlns="http://www.w3.org/2000/svg" id="graph" width="640" height="480" style="left: 0px;"><title>multiple sclerosis → retina layer formation</title><desc>Graph visualization of the connectivity between multiple sclerosis (Disease) and retina layer formation (Biological Process).

<!-- Created at https://het.io/search/?source=2170&target=13008&metapaths=DaG<rGpBP%2CDaGcGpBP%2CDaGiGpBP%2CDaGpBP%2CDaGr>GpBP%2CDdG<rGpBP%2CDdGcGpBP%2CDdGiGpBP%2CDdGpBP%2CDdGr>GpBP%2CDlAdGpBP%2CDlAeGpBP%2CDlAuGpBP%2CDpCbGpBP%2CDpCdGpBP%2CDpCuGpBP%2CDrDaGpBP%2CDrDdGpBP%2CDrDuGpBP%2CDtCbGpBP%2CDtCdGpBP%2CDtCuGpBP%2CDuG<rGpBP%2CDuGcGpBP%2CDuGiGpBP%2CDuGpBP%2CDuGr>GpBP&complete= -->

This subgraph of Hetionet v1.0 was created from paths between the specified source/target nodes that occurred more than expected by chance. See https://het.io for more information.</desc><defs><marker id="graph_arrowhead" viewBox="0 0 100 100" refX="80" refY="50" orient="auto" markerUnits="userSpaceOnUse" markerWidth="10" markerHeight="10"><path d=" M 0 0 L 100 50 L 0 100 " fill="#424242"></path></marker></defs><g id="graph_view" transform="translate(298.33058287432556,222.57655125722457) scale(1.6684331532074523)"><g id="graph_contents"><g id="graph_edge_line_highlight_layer"></g><g id="graph_node_circle_highlight_layer"></g><g id="graph_edge_line_layer"><path class="graph_edge_line" marker-end="" fill="none" stroke="#424242" stroke-width="2" d="M -122.05593468917473 7.205306771433222 L -73.16377221649225 17.072780366973525" style="stroke-dasharray: none;"></path><path class="graph_edge_line" marker-end="" fill="none" stroke="#424242" stroke-width="2" d="M 39.15501458989584 21.69282700283762 L -24.15305628182972 21.891472102766144" style="stroke-dasharray: none;"></path><path class="graph_edge_line" marker-end="" fill="none" stroke="#424242" stroke-width="2" d="M 88.15186155351432 16.65059309027375 L 148.0455727031875 4.387335867394157" style="stroke-dasharray: none;"></path><path class="graph_edge_line" marker-end="" fill="none" stroke="#424242" stroke-width="2" d="M -126.39736967131184 16.998404366048078 L -69.33399730751032 59.07945816273735" style="stroke-dasharray: none;"></path><path class="graph_edge_line" marker-end="" fill="none" stroke="#424242" stroke-width="2" d="M -24.664721024258725 73.848039614374 L 38.481914348960544 74.04993922447987" style="stroke-dasharray: none;"></path><path class="graph_edge_line" marker-end="" fill="none" stroke="#424242" stroke-width="2" d="M 83.6506536290523 60.14220635524717 L 151.87367571744042 13.409626945292159" style="stroke-dasharray: none;"></path><path class="graph_edge_line" marker-end="" fill="none" stroke="#424242" stroke-width="2" d="M -121.7637270143157 -0.8072049598445776 L -13.715885421840827 -14.520146167832356" style="stroke-dasharray: none;"></path><path class="graph_edge_line" marker-end="" fill="none" stroke="#424242" stroke-width="2" d="M 35.45015609950936 -15.035744864470544 L 147.67954542917786 -3.1778013267973937" style="stroke-dasharray: none;"></path><path class="graph_edge_line" marker-end="" fill="none" stroke="#424242" stroke-width="2" d="M -71.44371039840144 -40.78001852469376 L -124.86541928024846 -10.036162308462725" style="stroke-dasharray: none;"></path><path class="graph_edge_line" marker-end="" fill="none" stroke="#424242" stroke-width="2" d="M -25.242368250894028 -53.14850654929805 L 50.16064797920254 -53.21970436072587" style="stroke-dasharray: none;"></path><path class="graph_edge_line" marker-end="" fill="none" stroke="#424242" stroke-width="2" d="M 96.68085073324613 -41.46936216027046 L 150.52232771668102 -12.350950926126096" style="stroke-dasharray: none;"></path></g><g id="graph_edge_label_layer"><text class="graph_edge_label" font-size="8" font-weight="500" text-anchor="middle" user-select="none" fill="#424242" x="0" y="0" dy="-2.8" transform="translate(-97.60985345283349,12.139043569203373) rotate(11.410229498797838) " style="cursor: pointer;">associates</text><text class="graph_edge_label" font-size="8" font-weight="500" text-anchor="middle" user-select="none" fill="#424242" x="0" y="0" dy="-2.8" transform="translate(7.500979154033061,21.79214955280188) rotate(-0.17977942366761113) " style="cursor: pointer;">interacts</text><text class="graph_edge_label" font-size="8" font-weight="500" text-anchor="middle" user-select="none" fill="#424242" x="0" y="0" dy="-2.8" transform="translate(118.09871712835091,10.518964478833952) rotate(-11.571397529343638) " style="cursor: pointer;">participates</text><text class="graph_edge_label" font-size="8" font-weight="500" text-anchor="middle" user-select="none" fill="#424242" x="0" y="0" dy="-2.8" transform="translate(-97.86568348941108,38.038931264392716) rotate(36.406704253081195) " style="cursor: pointer;">associates</text><text class="graph_edge_label" font-size="8" font-weight="500" text-anchor="middle" user-select="none" fill="#424242" x="0" y="0" dy="-2.8" transform="translate(6.908596662350909,73.94898941942694) rotate(0.18319196354820674) " style="cursor: pointer;">interacts</text><text class="graph_edge_label" font-size="8" font-weight="500" text-anchor="middle" user-select="none" fill="#424242" x="0" y="0" dy="-2.8" transform="translate(117.76216467324636,36.775916650269664) rotate(-34.411034365108215) " style="cursor: pointer;">participates</text><text class="graph_edge_label" font-size="8" font-weight="500" text-anchor="middle" user-select="none" fill="#424242" x="0" y="0" dy="-2.8" transform="translate(-67.73980621807827,-7.663675563838467) rotate(-7.233049936898174) " style="cursor: pointer;">associates</text><text class="graph_edge_label" font-size="8" font-weight="500" text-anchor="middle" user-select="none" fill="#424242" x="0" y="0" dy="-2.8" transform="translate(91.56485076434362,-9.106773095633969) rotate(6.031385314228411) " style="cursor: pointer;">participates</text><text class="graph_edge_label" font-size="8" font-weight="500" text-anchor="middle" user-select="none" fill="#424242" x="0" y="0" dy="-2.8" transform="translate(-98.15456483932495,-25.40809041657824) rotate(-29.920155124431858) " style="cursor: pointer;">resembles</text><text class="graph_edge_label" font-size="8" font-weight="500" text-anchor="middle" user-select="none" fill="#424242" x="0" y="0" dy="-2.8" transform="translate(12.459139864154254,-53.18410545501196) rotate(-0.05410039408233782) " style="cursor: pointer;">upregulates</text><text class="graph_edge_label" font-size="8" font-weight="500" text-anchor="middle" user-select="none" fill="#424242" x="0" y="0" dy="-2.8" transform="translate(123.60158922496358,-26.910156543198276) rotate(28.405297017185507) " style="cursor: pointer;">participates</text></g><g id="graph_node_circle_layer"><circle class="graph_node_circle" r="25" fill="#02b3e4" stroke="none" stroke-width="2" transform="translate(-48.90293444472947,21.969131111375802)" style="stroke-dasharray: 4, 2; cursor: pointer;"></circle><circle class="graph_node_circle" r="25" fill="#02b3e4" stroke="none" stroke-width="2" transform="translate(63.90489275279559,21.615167994227964)" style="stroke-dasharray: 4, 2; cursor: pointer;"></circle><circle class="graph_node_circle" r="25" fill="#02b3e4" stroke="none" stroke-width="2" transform="translate(-49.41459451788465,73.76890650175449)" style="stroke-dasharray: 4, 2; cursor: pointer;"></circle><circle class="graph_node_circle" r="25" fill="#02b3e4" stroke="none" stroke-width="2" transform="translate(63.23178784258647,74.12907233709939)" style="stroke-dasharray: 4, 2; cursor: pointer;"></circle><circle class="graph_node_circle" r="25" fill="#02b3e4" stroke="none" stroke-width="2" transform="translate(10.837160024780964,-17.63630715470788)" style="stroke-dasharray: 4, 2; cursor: pointer;"></circle><circle class="graph_node_circle" r="25" fill="#875442" stroke="none" stroke-width="2" transform="translate(-49.992357217712396,-53.125136860187425)" style="stroke-dasharray: 4, 2; cursor: pointer;"></circle><circle class="graph_node_circle" r="25" fill="#02b3e4" stroke="none" stroke-width="2" transform="translate(74.9106369460209,-53.243074049836494)" style="stroke-dasharray: 4, 2; cursor: pointer;"></circle><circle class="graph_node_circle" r="25" fill="#875442" stroke="none" stroke-width="2" transform="translate(-146.3167724609375,2.308956027030945)" style="stroke-dasharray: 4, 2; cursor: pointer;"></circle><circle class="graph_node_circle" r="25" fill="#fa750f" stroke="none" stroke-width="2" transform="translate(172.29254150390625,-0.5772390365600586)" style="stroke-dasharray: 4, 2; cursor: pointer;"></circle></g><g id="graph_node_label_layer"><foreignObject class="graph_node_label" x="-25" y="-25" width="50" height="50" transform="translate(-48.90293444472947,21.969131111375802)" style="pointer-events: none;"><div style="display: flex; justify-content: center; align-items: center; width: 100%; height: 100%; box-sizing: border-box; padding: 6.25px;"><span style="text-align: center; font-size: 8px; font-weight: 500; color: rgb(250, 250, 250); word-break: break-word;">DKKL1</span></div></foreignObject><foreignObject class="graph_node_label" x="-25" y="-25" width="50" height="50" transform="translate(63.90489275279559,21.615167994227964)" style="pointer-events: none;"><div style="display: flex; justify-content: center; align-items: center; width: 100%; height: 100%; box-sizing: border-box; padding: 6.25px;"><span style="text-align: center; font-size: 8px; font-weight: 500; color: rgb(250, 250, 250); word-break: break-word;">FJX1</span></div></foreignObject><foreignObject class="graph_node_label" x="-25" y="-25" width="50" height="50" transform="translate(-49.41459451788465,73.76890650175449)" style="pointer-events: none;"><div style="display: flex; justify-content: center; align-items: center; width: 100%; height: 100%; box-sizing: border-box; padding: 6.25px;"><span style="text-align: center; font-size: 8px; font-weight: 500; color: rgb(250, 250, 250); word-break: break-word;">PTPRK</span></div></foreignObject><foreignObject class="graph_node_label" x="-25" y="-25" width="50" height="50" transform="translate(63.23178784258647,74.12907233709939)" style="pointer-events: none;"><div style="display: flex; justify-content: center; align-items: center; width: 100%; height: 100%; box-sizing: border-box; padding: 6.25px;"><span style="text-align: center; font-size: 8px; font-weight: 500; color: rgb(250, 250, 250); word-break: break-word;">PTPRM</span></div></foreignObject><foreignObject class="graph_node_label" x="-25" y="-25" width="50" height="50" transform="translate(10.837160024780964,-17.63630715470788)" style="pointer-events: none;"><div style="display: flex; justify-content: center; align-items: center; width: 100%; height: 100%; box-sizing: border-box; padding: 6.25px;"><span style="text-align: center; font-size: 8px; font-weight: 500; color: rgb(250, 250, 250); word-break: break-word;">AHI1</span></div></foreignObject><foreignObject class="graph_node_label" x="-25" y="-25" width="50" height="50" transform="translate(-49.992357217712396,-53.125136860187425)" style="pointer-events: none;"><div style="display: flex; justify-content: center; align-items: center; width: 100%; height: 100%; box-sizing: border-box; padding: 6.25px;"><span style="text-align: center; font-size: 8px; font-weight: 500; color: rgb(250, 250, 250); word-break: break-word;">amyotrophic lateral sc...</span></div></foreignObject><foreignObject class="graph_node_label" x="-25" y="-25" width="50" height="50" transform="translate(74.9106369460209,-53.243074049836494)" style="pointer-events: none;"><div style="display: flex; justify-content: center; align-items: center; width: 100%; height: 100%; box-sizing: border-box; padding: 6.25px;"><span style="text-align: center; font-size: 8px; font-weight: 500; color: rgb(250, 250, 250); word-break: break-word;">TOPORS</span></div></foreignObject><foreignObject class="graph_node_label" x="-25" y="-25" width="50" height="50" transform="translate(-146.3167724609375,2.308956027030945)" style="pointer-events: none;"><div style="display: flex; justify-content: center; align-items: center; width: 100%; height: 100%; box-sizing: border-box; padding: 6.25px;"><span style="text-align: center; font-size: 8px; font-weight: 500; color: rgb(250, 250, 250); word-break: break-word;">multiple sclerosis</span></div></foreignObject><foreignObject class="graph_node_label" x="-25" y="-25" width="50" height="50" transform="translate(172.29254150390625,-0.5772390365600586)" style="pointer-events: none;"><div style="display: flex; justify-content: center; align-items: center; width: 100%; height: 100%; box-sizing: border-box; padding: 6.25px;"><span style="text-align: center; font-size: 8px; font-weight: 500; color: rgb(250, 250, 250); word-break: break-word;">retina layer formation</span></div></foreignObject></g></g></g></svg>
```

</details>

I see the node names in the source, but they don't show up.

**Comment by dhimmel** [comment]
Unrelated but should the following be inside an HTML comment or some type of element:

> This subgraph of Hetionet v1.0 was created from paths between the specified source/target nodes that occurred more than expected by chance. See https://het.io for more information.

**Comment by vincerubinetti** [comment]
Closed by #156 

> Unrelated but should the following be inside an HTML comment or some type of element:

It is within an element: the `<desc>` element. There can be plain text inside that element, just like a `<text>` element. The comment only surrounds the URL to not throw off the parser with those special characters. However, I think it looks kinda weird and arbitrary to have the comment only around the URL line, so I made the comment around the whole description within the `<desc>` tag.

**Comment by dhimmel** [comment]
Following https://github.com/greenelab/connectivity-search-frontend/pull/156 node labels show up in my browser but not on my desktop The GNOME image viewer app. This should be sufficient for my presentation.
